.github/workflows/publish.yaml

```diff
@@ -4,6 +4,8 @@ on:
   workflow_dispatch:  # This line adds the manual trigger
   push:
     branches: [ "main" ]
+  pull_request: # this should be removed before merging
```

Comment:
Don't forget about this I guess. Was it just for testing?

Reply:
Indeed, it was used for testing, it is removed now. Could you please approve the PR again?